pkgs/by-name/sd/SDL2_Pango/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, freetype
+, pango
+, SDL2
+, darwin
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sdl2-pango";
+  version = "2.1.5";
+
+  src = fetchFromGitHub {
+    owner = "markuskimius";
+    repo = "SDL2_Pango";
+    rev = "v${version}";
+    hash = "sha256-8SL5ylxi87TuKreC8m2kxlLr8rcmwYYvwkp4vQZ9dkc=";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    autoreconfHook
```

Does it work with `strictDeps = true;`?

```suggestion
    SDL2
    autoreconfHook
```